### DISCUSSION TITLE
Configuration of HMDI audio extractor and Dexed compliance

### DISCUSSION THREAD

**Discussion by NickHowes** [probonopd/MiniDexed#209] [Q&A] [answered]
Two questions if I may

I have tried to use a Link4 HDMI audio extractor on a PI3B+ running Mini DEXED with no joy (tried both options in the config file). Are there any known issues with any of the HDMI extractors or additional steps to set up?

https://www.amazon.co.uk/gp/product/B08T94T8J5/ref=ppx_yo_dt_b_asin_title_o01_s00?ie=UTF8&psc=1  is the HDMI extractor I tested

Second, is there a plan to support direct output from DEXED to the PI (over MIDI) for voice dumps (Cart > DX7) Saving as .syx files is obviously one option, but it'd be great to have a real-time editing capability, especially as it would instantly free up CPU load inside the DAW, by editing/eval a sound in Dexed and then literally offloading it straight to the PI via a MIDI dump

Many thanks for any answers

**Comment by probonopd** [comment]
Hello @NickHowes. Unfortunately I have no experience with that audio extractor. Do you get sound when you connect MiniDexed to, e.g., a TV with HDMI?

**Reply by NickHowes** [reply]
I do indeed, connected to my Samsung's TV HDMI and it works fine

**Reply by sou7611** [reply]
It's possible you might need an EDID box to "wake up" the HDMI output.

Something like this : https://www.amazon.com/dp/B09NPWKQYC

**Comment by probonopd** [comment, marked as answer]
> is there a plan to support direct output from DEXED to the PI (over MIDI) for voice dumps (Cart > DX7)

This is being worked on currently: https://github.com/probonopd/MiniDexed/issues/82

**Reply by NickHowes** [reply]
Excellent, thank you!
